src/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.cpp

```diff
@@ -130,6 +130,11 @@ VectorXd Tabulated::computeContribution(
         throw ostk::core::error::runtime::Wrong("Frame");
     }
 
+    if (anInstant < instants_.accessFirst() || anInstant > instants_.accessLast())
+    {
+        return VectorXd::Zero(interpolators_.getSize());
+    }
+
     const double epoch = (anInstant - instants_.accessFirst()).inSeconds();
```

Comment:
```suggestion
    const Real epoch = (anInstant - instants_.accessFirst()).inSeconds();
```

Reply:
The evaluator accepts a `double` https://github.com/open-space-collective/open-space-toolkit-mathematics/blob/93d7dc56e59d906c9c5527e828ce36f2f3a2225f/src/OpenSpaceToolkit/Mathematics/CurveFitting/Interpolator.cpp#L18 so it think it's better to be explicit.

Reply:
Ok to keep it like this for now, but in the future perhaps better to align OSTk with using `Real` whenever possible?

Reply:
Fair, I think we can update the interface there to accept a Real and then convert to a double when interfacing with boost.